website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,207 @@
+!!!
+
+!!! Do
+
+If it's necessary to persist the Rich Tooltip within the UI, consider using a `secondary` [Button](/components/button) as the toggle element. In this scenario use the [`on click`](#interaction) interaction to persist the component.
+
+![Example using a Button to toggle the component](/assets/components/rich-tooltip/rich-tooltip-trigger-secondary-button.png)
+
+This can be useful when displaying instructional information or details that reference different parts of the UI.
+!!!
+
+!!! Dont
+
+Don’t use a [Button](/components/button) to toggle a Rich Tooltip if the Button performs a function. This can cause an unexpected overlap in the interaction with the Button and the toggleing of the Rich Tooltip.
+
+![Example of incorrectly toggling a Rich Tooltip with a button](/assets/components/rich-tooltip/rich-tooltip-trigger-button-dont.png)
+!!!
+
+## Interaction
+
+The Rich Tooltip supports two different interaction methods: a `hover/focus` ("soft") interaction and a `click` interaction. Which one to use depends on the type and complexity of content displayed by the component, and whether _persisting_ the content in the UI is necessary or beneficial to the user.
+
+1. `Hover/focus` (default): displays when the mouse enters the toggle or the toggle receives focus. The `RichTooltip` has a timeout by default and will persist for 500 milliseconds (0.5 seconds) _after_ the mouse leaves the toggle.
+2. `On click`: displays when the user clicks the toggle with a mouse or if the toggle receives a keyboard event (`spacebar`, `enter/return`).
+
+## Content
+
+We are not prescriptive about what type of content a Rich Tooltip can contain; consider these common examples when determining whether the Rich Tooltip is appropriate for your use case.
+
+<!-- Ensure that we have alignment for all of these scenarios -->
+
+### Text with links
+
+This is a common use case for a Rich Tooltip due to the lack of support for interactive content in the standard [Tooltip](/components/tooltip).
+
+![Link within text in the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-links.png)
+
+### Structured content
+
+The Rich Tooltip supports more structure and layout methods to better organize the content hierarchically.
+
+![Structured content within the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-structure.png)
+
+### Key-value pairs
+
+Use the Rich Tooltip to display supplemental key-value pairs or metadata related to an element.
+
+![Key-value pairs within the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-key-value-pair.png)
+
+### Overflow content
+
+![Displaying overflow content with the Rich Tooltip](/assets/components/rich-tooltip/rich-tooltip-content-overflow.png)
+
+### Onboarding and walkthrough
```

Comment:
Thinking on this point that we spoke about, I think we should just remove these sections entirely rather than have a section for "Future use cases". Could be a bit confusing for consumers and might do more harm than good.

Reply:
for now I'll comment it out, we can determine whether it's useful at a later point.